test/markdown_test.dart

```diff
@@ -18,30 +18,46 @@ void main() {
 
   group('Resolver', () {
     Node nyanResolver(String text) => new Text('~=[,,_${text}_,,]:3');
-    validateCore('simple link resolver', '''
+    validateCore(
```

Comment:
Oops, I forgot we don't dartfmt this file. Maybe we should just leave it, even though it doesn't have great readability? (To be clear, the previous formatting isn't much better.)